setup/aws-iot-device-client.service

```diff
@@ -4,7 +4,7 @@ Wants=network-online.target
 After=network.target network-online.target
 
 [Service]
-Environment="CONF_PATH=/etc/.aws-iot-device-client/aws-iot-device-client.conf"
```

Comment:
The path is correct. We intentionally create device client resources in an hidden file 

https://github.com/awslabs/aws-iot-device-client/blob/f207eafc31b723492e33d5941fe1d05ba46c59b8/docs/SETUP.md?plain=1#L29